Fixes
iconvconfig.c: In function 'write_output':
iconvconfig.c:973: error: invalid storage class for function 'name_insert'

--- glibc-2.2.5/iconv/iconvconfig.c.old	2002-01-20 19:20:53.000000000 -0800
+++ glibc-2.2.5/iconv/iconvconfig.c	2005-03-11 14:30:27.000000000 -0800
@@ -969,7 +969,12 @@
   char tmpfname[sizeof (GCONV_MODULES_CACHE) + strlen (".XXXXXX")];
 
   /* Function to insert the names.  */
-  static void name_insert (const void *nodep, VISIT value, int level)
+#if __GNUC__ >= 4
+  auto 
+#else
+  static 
+#endif
+  void name_insert (const void *nodep, VISIT value, int level)
     {
       struct name *name;
       unsigned int idx;
